devdox/app/services/git_tokens_service.py

```
@@ -0,0 +1,60 @@
+
+from fastapi import Depends
+
+from app.repositories.git_label_repository import TortoiseGitLabelStore
+from app.schemas.basic import RequiredPaginationParams
+
+
+class GetGitLabelService:
+
+    def __init__(
+            self,
+            label_store: TortoiseGitLabelStore = Depends()
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)